language-support/ts/codegen/src/TsCodeGenMain.hs

```diff
@@ -743,9 +743,10 @@ setupWorkspace optInputPackageJson optOutputDir dependencies = do
     (Just <$> BSL.readFile optInputPackageJson) (const $ pure Nothing)
   packageJson <- case mbBytes of
     Nothing -> pure mempty
-    Just bytes -> case decode @PackageJson bytes of
-      Nothing -> fail $ "Error decoding JSON from '" <> optInputPackageJson <> "'"
-      Just packageJson -> pure packageJson
+    Just bytes ->
+      case eitherDecode @PackageJson bytes of
+        Left msg -> fail $ "Error : '" <> optInputPackageJson <> "' : " <> msg
```

Comment:
You've apparently done too much OCaml in your life. Nobody else would leave a space between "Error" and the colon afterwards. 😉